docs/content/docs/dev/table/sql/create.md

```diff
@@ -616,6 +616,7 @@ INSERT INTO my_rtas_table SELECT id, name, age FROM source_table WHERE mod(id, 1
 * Does not support specifying primary key constraints yet.
 
 **Note:** By default, RTAS is non-atomic which means the table won't be dropped or restored to its origin automatically if occur errors while inserting data into the table.
+**Note:** In the InMemoryCatalog scenario, the drop table only removes the metadata and does not clean up the underlying data, when reusing RTAS, it is recommended to make sure that the underlying data has been cleaned up first, otherwise it may look like the data is duplicated.
```

Comment:
**Note:** If using the in-memory catalog, the drop table only removes the metadata, not the data in physical table. Therefore user needs to ensure that the data is cleaned up first, otherwise, it will lead to data duplication.

Reply:
thanks for your review!  The root cause is not clear enough if it's executed via sql-client, so we want to be able to tell the user that this DynamicTableSink doesn't support atomicity and needs to be turned off.
See [FLINK-32787](https://issues.apache.org/jira/browse/FLINK-32787) for details.